config.json

```diff
@@ -1,1052 +1,1065 @@
 {
-  "language": "ECMAScript",
   "active": true,
-  "test_pattern": ".*[.]spec[.]js$",
   "exercises": [
     {
-      "uuid": "9ce0f408-6d7b-4466-a390-75aeaf9492f2",
-      "slug": "hello-world",
       "core": true,
       "difficulty": 1,
+      "slug": "hello-world",
       "topics": [
-        "Control-flow (conditionals)",
-        "Optional values",
-        "Strings",
-        "Text formatting"
-      ]
+        "control-flow-(conditionals)",
+        "optional-values",
+        "strings",
+        "text-formatting"
+      ],
+      "uuid": "9ce0f408-6d7b-4466-a390-75aeaf9492f2"
     },
     {
-      "uuid": "7c8294ee-5924-4bf8-a72f-31d0e2d7d9a0",
-      "slug": "leap",
       "core": true,
       "difficulty": 1,
+      "slug": "leap",
       "topics": [
-        "Booleans",
-        "Integers",
-        "Logic"
-      ]
+        "booleans",
+        "integers",
+        "logic"
+      ],
+      "uuid": "7c8294ee-5924-4bf8-a72f-31d0e2d7d9a0"
     },
     {
-      "uuid": "342974d6-9083-4754-a6c5-ed1e19e40ec5",
-      "slug": "rna-transcription",
       "core": true,
       "difficulty": 1,
+      "slug": "rna-transcription",
       "topics": [
-        "Strings",
-        "Transforming"
-      ]
+        "strings",
+        "transforming"
+      ],
+      "uuid": "342974d6-9083-4754-a6c5-ed1e19e40ec5"
     },
     {
-      "uuid": "62d60b42-93bc-4de9-90d1-1ca18a847812",
-      "slug": "simple-cipher",
       "core": true,
       "difficulty": 1,
+      "slug": "simple-cipher",
       "topics": [
-        "Control-flow (conditionals)",
-        "Control-flow (loops)",
-        "Algorithms",
-        "Mathematics",
-        "Strings",
-        "Randomness",
-        "Text formatting",
-        "Transforming"
-      ]
+        "algorithms",
+        "control-flow-(conditionals)",
+        "control-flow-(loops)",
+        "mathematics",
+        "randomness",
+        "strings",
+        "text-formatting",
+        "transforming"
+      ],
+      "uuid": "62d60b42-93bc-4de9-90d1-1ca18a847812"
     },
     {
```

Comment:
I put it here, because it is related to the rna-transcriptor. That's why I let it unlock with that exercise.
The uuid I generated with https://www.uuidgenerator.net